### PR TITLE
Updated read.me to correct for mistake found on poster

### PR DESCRIPTION
**Pull request by ChristopherDavidDean**
Just a quick update to the read.me to account for the mistake Dan found on the poster. Didn't want to just push direct to main, so have wacked it as a quick branch, hope that's okay. Just need two of you to sign off on this!

### CODE REVIEW

**Comment by bethany-j-allen** [comment]
Thanks @ChristopherDavidDean! This mistake doesn't transfer to the matching vignette so I think it should be the only needed change.

**Comment by willgearty** [comment]
This change dramatically changes the plot from this:
image

to this:
image

This new result (dramatically higher carbonates in Paleozoic) seems weird to me?

**Comment by bethany-j-allen** [comment]
Yes, that is why Dan noticed, he said that our original plot is incorrect.

**Comment by ChristopherDavidDean** [comment]
> This change dramatically changes the plot from this: image
> 
> to this: image
> 
> This new result (dramatically higher carbonates in Paleozoic) seems weird to me?

Yeah I had a compare of the graphs, and whilst it's a dramatic change it makes a lot more sense to me. Fundamentally N. America has been basically a large continental landmass since the Triassic, with only a few epicontinental seas popping up in the Jurassic/Cretaceous, both of which have few carbonate units that cover wide geographic and temporal ranges (e.g. The Greenhorn Limestone in the WIS covers a big chunk of the Cenomanian/Turonian and also the majority of the West of the USA, but is just 1 counted unit here). So whilst it's a big drop off, it kinda makes sense. Below I've done the same thing, but for siliciclastic units:

image

Which also doesn't look too outrageous to me. Honestly I think another big issue is probably just quality of the data; a lot of sedimentary units are listed as having both carbonate and siliciclastic elements, so that might also be a factor. I know we want to have a result that makes sense, but the previous graph made even less sense, and ultimately this is a better reflection of the data held in Macrostrat. Alternatively, we could avoid the issues related to lithology and just use a specific environment instead?

**Comment by willgearty** [comment]
Oh right, we're only covering North America here...Looking at Peters and Heim 2010, this result is pretty similar to theirs, although we use "carbonate" instead of "marine". Maybe we should switch to that to be more comparable?

**Comment by bethany-j-allen** [comment]
That is specifically what is done in the vignette :) so I don't know whether it matters for this code, which is currently only in the readme? What's the goal here?

**Comment by willgearty** [comment]
I think mirroring that in the README example would be good.

**Comment by ChristopherDavidDean** [comment]
Yeah these are fair comments. Here's what it looks like set to marine:

image

I'll go ahead and update it now.

**Comment by ChristopherDavidDean** [comment]
> That looks great, thanks @ChristopherDavidDean! Make sure to also update the image file that is linked in the README.

Done! Updated the figure and the direction to it, as well as catching that I'd not updated the y-axis title. This should all be good now.

**Review by willgearty — APPROVED** [review]
Beautiful, thanks!

File: README.md [hunk 1]
```diff
@@ -93,33 +93,33 @@ ggplot() +
...
 
-A minimal example of getting and plotting the number of carbonate units through time for North America:
+A minimal example of getting and plotting the number of marine units through time for North America:
 
 ```r
 # Load libraries
 library(rmacrostrat)
 library(ggplot2)
 library(deeptime)
 # Get all carbonate units for North America
```
Comment (LewisAJones) [inline README.md:103]:
```suggestion
# Get all marine units for North America
```

**Review by LewisAJones — APPROVED** [review]
Looks good - tiny update to comments needed!

**Pull request merged**